rosco-web/src/main/groovy/com/netflix/spinnaker/rosco/controllers/BakeryController.groovy

```diff
@@ -134,11 +135,9 @@ class BakeryController {
     }
 
     // Ok, it didn't fail right away; the bake is underway.
-    BakeStatus returnedBakeStatus = bakeStore.storeNewBakeStatus(bakeKey,
-                                                                 region,
-                                                                 bakeRequest,
-                                                                 newBakeStatus,
-                                                                 jobRequest.tokenizedCommand.join(" "))
+    BakeStatus returnedBakeStatus = bakeStore.storeNewBakeStatus (
```

Comment:
Don't see the need to reformat here if just adding the one arg. I can live with it, but leads to more reading in review. Either way, don't need the space before the opening parens.